cabal-install/Distribution/Client/ProjectOrchestration.hs

```diff
@@ -608,7 +616,7 @@ reportBuildFailures verbosity plan buildOutcomes
 
        -- For all failures, print either a short summary (if we showed the
        -- build log) or all details
-       die $ unlines
+       notice verbosity $ unlines
```

Comment:
BTW, the current test failure is due to the same reason, looking for the expected SIGSEGV in the stderr rather than stdout.

@23Skidoo what's our policy on this kind of thing?